<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCardDialog</class>
 <widget class="QDialog" name="AddCardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddCardDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(35, 39, 42);

</string>
  </property>
  <widget class="QPushButton" name="addCardPushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Whitney</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
     background-color: rgb(88, 101, 242);
 }
 QPushButton#pushButton:pressed {
   	background-color: rgb(88, 157, 242); 
 }
 QPushButton#pushButton:hover {
     
	background-color: rgb(88, 157, 242);
 }
 QPushButton[text] {
    color:white;
 }</string>
   </property>
   <property name="text">
    <string>Add </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardNameLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Whitney</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit[text]{
color: white;}
QLineEdit{
background-color: rgb(44, 47, 51);
border:none;}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Card name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardHolderLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Whitney</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit[text]{
color: white;}
QLineEdit{
background-color: rgb(44, 47, 51);
border:none;}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>CARD HOLDER - CAPITAL LETTERS ONLY</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardNumberLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Whitney</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit[text]{
color: white;}
QLineEdit{
background-color: rgb(44, 47, 51);
border:none;}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Card number</string>
   </property>
  </widget>
  <widget class="QLabel" name="validUntilLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Whitney</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel[text]{
color: white;}
QLabel{
background-color:rgb(35, 39, 42);
border:none;}</string>
   </property>
   <property name="text">
    <string>Valid until:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="isoCurrencyComboBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Whitney</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
color:white;
background-color: rgb(44, 47, 51);
}

QComboBox QAbstractItemView {
   color:white;
    selection-background-color:rgb(88, 101, 242);
	backgroud-color:rgb(44, 47, 51);
	
}

QComboBox QScrollBar
{
background: #5865F2
}

QComboBox QAbstractItemView QScrollBar
{
background: #5865F2
}



</string>
   </property>
  </widget>
  <widget class="QLabel" name="currencyLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Whitney</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel[text]{
color: white;}
QLabel{
background-color:rgb(35, 39, 42);
border:none;}</string>
   </property>
   <property name="text">
    <string>Currency:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="validUntilLineEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Whitney</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit[text]{
color: white;}
QLineEdit{
background-color: rgb(44, 47, 51);
border:none;}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>MM/YY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelAddCardPushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Whitney</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
     background-color: rgb(88, 101, 242);
 }
 QPushButton#pushButton:pressed {
   	background-color: rgb(88, 157, 242); 
 }
 QPushButton#pushButton:hover {
     
	background-color: rgb(88, 157, 242);
 }
 QPushButton[text] {
    color:white;
 }</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelAddCardPushButton</sender>
   <signal>clicked()</signal>
   <receiver>AddCardDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
